docs/examples/MarrmotM14.ipynb

```diff
@@ -2,204 +2,157 @@
  "cells": [
```

Comment:
<p>Can we also print the cfg_file? Now I am unsure what the initialize call gets as an argument.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
yup, it is added.